api/client/profile.go

```diff
@@ -228,6 +234,8 @@ func ProfileFromFile(filePath string) (*Profile, error) {
 	return cp, nil
 }
 
+// SaveToDir saves Profile to the target directory.
+// if makeCurrent is true, attempt to make it the current profile.
```

Comment:
```suggestion
// if makeCurrent is true, make this profile current.
```
otherwise it reads as if the function itself is flaky.